src/App.css

```diff
@@ -0,0 +1,67 @@
+* {
+  margin: 0;
+ }
+ .app {
+  background-color: #fafafa;
+  height: 130vh;
+  
+}
+.component6 {
+  display: flex;
+  flex-direction: column;
+  position:absolute;
+  border: 1px solid lightgray;
+  top:100%;
+  right:50px;
+  transform:translateY(-130%);
+  max-width: 500px;
+  padding: 10px;
+}
+
+.title {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 10px;
+}
+.date {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 10px;
+}
+
+.players {
+  object-fit: contain;
+  padding: 20px;
+}
+
+.asuLogo {
+  padding: 10px;
+  height: 75px;
+  float: left;
+}
+
+.headerContainer {
+  height: 200px;
+  display: flex inline-block;
+  position: relative;
+}
+
+.headerText {
+  margin: 0;
+  width: 30%;
+  color: rgb(164, 0, 70);
+  position: absolute;
+  top: 10%;
+  left: 35%;
+  text-align: center;
+  text-shadow: 3px 3px 0px rgba(255, 196, 37, 0.4);
+}
+
+.headerRule {
+  height: 12px;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  margin: 0;
+  box-shadow: inset 0 12px 12px -12px rgba(164, 0, 70, 0.9);
+}
```

Comment:
Personally, i think we should look into finding a css in js solution ([styles-ss](https://styled-components.com/) or [emotion](https://emotion.sh/docs/introduction are good calls) This will let us remove having to keep css extra files and allow us to modularize our css more (each component will maintain it's own css and won't affect any other components).  Besides component specific styles, we should be able to get away with this file being a simple call to css normalize https://github.com/csstools/postcss-normalize/blob/master/INSTALL.md#create-react-app

Reply:
and make it easier to review the css itself without worrying about side effects, as it'll be tied closer to the component

Reply:
At least for now, per the poll we will keep the css separated.  We can discuss and look at changing it in the future.